### PR TITLE
Year dropdown implementation

### PR DESCRIPTION
**Pull request by arahansen**
This Pull Request would close issue/feature request #90.

I've written some tests that cover the new functionality. I'm fairly new to submitting pull requests for open source projects. So let me know if there's any improvements I can make or additional test coverage you'd like to see.

### CODE REVIEW

File: src/stylesheets/datepicker.scss [hunk 1]
```diff
@@ -150,6 +177,94 @@
...
+  width: 50%;
+  left: 25%;
+  position: absolute;
+  bottom: 25px;
+  border: 1px solid transparent;
+  border-radius: $border-radius;
+
+  &:hover {
+    cursor: pointer;
+    // background-color: darken($background-color, 10%);
```
Comment (martijnrusschen) [inline src/stylesheets/datepicker.scss]: You can remove these.

File: src/stylesheets/datepicker.scss [hunk 2]
```diff
@@ -150,6 +177,94 @@
...
+  margin-left: auto;
+  margin-right: auto;
+
+  &:first-of-type {
+    border-top-left-radius: $border-radius;
+    border-top-right-radius: $border-radius;
+  }
+
+  &:last-of-type {
+    -webkit-user-select: none;
```
Comment (martijnrusschen) [inline src/stylesheets/datepicker.scss]: I should really implement an auto-prefixer for CSS

Comment (RSO) [inline src/stylesheets/datepicker.scss]: Yeah, probably good to pick up in a separate PR though.

**Comment by martijnrusschen** [comment]
Thanks @arahansen for picking this up! Really appreciate the work you did here. And a big plus for adding tests for the new functionality! Please take a look at the comments and we're happy to merge this!

**Comment by arahansen** [comment]
Comments removed! :)

**Pull request merged**